File: src/Model/Application/SplitsResponse.cs
namespace Model.Application
{
    public class SplitsResponse
    {
        public IEnumerable<Split> Splits { get; set; }
        public SplitSummary SplitSummary { get; set; }
        public string ResponseInfo { get; set; }

        /* chart data */
        public IEnumerable<SplitChartItem> SplitChartItems { get; set; } 
    }
}

File: src/budgetwallettsapp/src/types/api/splitsResponse.ts
import { Split } from "./split";
import { SplitSummary } from "./splitSummary";
import { SplitChartItem } from "./splitChartItem";

export interface SplitsResponse {
    splits: Split[],
    splitSummary: SplitSummary,
    splitChartItems: SplitChartItem[],
    responseInFo: string,
}